doc/classes/Tree.xml

```diff
@@ -193,7 +193,7 @@
 			<argument index="1" name="min_width" type="int">
 			</argument>
 			<description>
-				Sets the minimum width of a column.
+				Sets the minimum width of a column. Columns that have the "Expand" flag will use their "min_width" in a similar fashion to the "stretch_ratio" size flag of [Control].
```

Comment:
You can use `[member Control.size_flags_stretch_ratio]` to refer to the latter directly with a hyperlink.